x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/general.ts

```diff
@@ -78,6 +78,7 @@ export default ({ getPageObjects, getPageObject, getService }: FtrProviderContex
       const createdAction = await createSlackConnector({
         name: connectorName,
         supertest,
+        getService,
```

Comment:
Following up from my previous comment about removing `supertest` as a prop, we'll just need to carry those changes through to here.

Reply:
Thanks! Fixed in bc960b34ee5c4c7b76fa0b8934a00725c4aca863

Reply:
Sorry what I meant was that line 80 in this file is no longer needed either. `createSlackConnector` doesn't use `supertest` anymore.